src/core/api/link_raw_api.cpp

```diff
@@ -67,6 +84,9 @@ otError otLinkRawSetPanId(otInstance *aInstance, uint16_t aPanId)
     VerifyOrExit(aInstance->mLinkRaw.IsEnabled(), error = OT_ERROR_INVALID_STATE);
 
     otPlatRadioSetPanId(aInstance, aPanId);
```

Comment:
In order not to block other PR, I think it will be better to submit another PR for radio-only task. This PR only splits `ncp_base` and adding namespace `ot::Ncp`.